README.md

```diff
@@ -1 +1,138 @@
-# MealDB
\ No newline at end of file
+<a name="readme-top"></a>
+![](https://img.shields.io/badge/Microverse-blueviolet)
+<div align="center">
+
+  <h3><b>Javascript Capstone Project</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Eat Healthy] <a name="about-project"></a>
+
+The Eat Healthy is a web application that utilizes an external API from 
+[https://www.themealdb.com/api/json/v1/1/filter.php?a=British] to showcase a list of healthy meals. Users have the ability to interact with the meals by liking them, leaving comments, or reserving a meal using an involvement API. 
+
+The objective of the project is to develop a dynamic web application that meets the standards of a professional web developer.
+## Project requirements 
+- [x] Use ES6 modules, with import and export.
+- [x] Use callbacks and promises.
+- [x] Use Webpack to bundle JavaScript.
+- [x] Write units tests for a JavaScript app.
+- [x] Follow Gitflow.
+- [x] Send and receive data from an API.
+- [x] Use API documentation.
+- [x] page counter.
+
+## Project presentation
+## Screenshot
+![screenshot](./screenshoot/eat%20healthy%20home%20page.png)
+![screenshot](./screenshoot/eat%20healthy%20popup.png)
+
+## Please take a look at the video below
+
+[Video Recording](#)
+## How it works
+
+The Meal DB web application allows users to like, comment, and make a reservation for their favorite meal.
+* By clicking on `like button` the number of like is increased and stored in the involvement API;
+* By clicking on the `Comment button` to display the description of the selected meal, the list of all comments, and added a new comment;
+
+
+## Live Demo
+
+[Live demo here](https://yosaddis.github.io/MealDB/)
```

Comment:
[Optional]
Instead of the current repository link, Please add a link to the deployed project here 👍